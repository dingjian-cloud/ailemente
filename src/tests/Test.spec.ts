import { shallowMount } from '@vue/test-utils';
import Demo from '@/components/Test.vue';
import { test, expect } from 'vitest';

test('Test.vue', async () => {
  const wrapper = shallowMount(Demo);
  expect(wrapper.html()).toContain('Unit Test Page');
  expect(wrapper.html()).toContain('count is: 0');
  await wrapper.find('button').trigger('click');
  expect(wrapper.html()).toContain('count is: 1');
  expect(wrapper.html()).toMatchSnapshot();
});
