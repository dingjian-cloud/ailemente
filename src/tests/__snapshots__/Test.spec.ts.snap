// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test.vue 1`] = `
"<div class=\\"test-container page-container\\">
  <div class=\\"page-title\\">Unit Test Page</div>
  <p>count is: 1</p><button>increment</button>
</div>"
`;
